<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INBNRMARITA" failures="2" tests="3" name="Reimbursement Approve By Manager" time="20.118" errors="0" timestamp="2023-05-29T16:25:10 IST">
  <testcase classname="script.ReimbApprovalByManager" name="validatePendingReimbursement" time="6.507">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [1070] but found [₹1070]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [1070] but found [₹1070]
at script.ReimbApprovalByManager.validatePendingReimbursement(ReimbApprovalByManager.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validatePendingReimbursement -->
  <testcase classname="script.ReimbApprovalByManager" name="validateReimApprovedPopup" time="0.852"/>
  <testcase classname="script.ReimbApprovalByManager" name="validateManagerApprovedReim" time="3.396">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [1070] but found [₹1070]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [1070] but found [₹1070]
at script.ReimbApprovalByManager.validateManagerApprovedReim(ReimbApprovalByManager.java:102)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- validateManagerApprovedReim -->
</testsuite> <!-- Reimbursement Approve By Manager -->
